server/models/dataset.py

```diff
@@ -390,7 +390,9 @@ def applyMetadata(self, dataset, metadataFile, save):
 
                     # Add updated image to list of images to potentially save
                     images.append(image)
-                except RowMetadataException as e:
+                except RowMetadataWarning as e:
```

Comment:
It's ok to use exception handling to get capture warnings in this case, since a failure to match in `_getImageForMetadataCsvRow` is still "fatal" and will abort processing of the whole row. In the future, I anticipate that we might have non-fatal warnings for in rows, which will need to be captured differently.